frontend/src/libs/helpers/get-number-of-days/get-number-of-days.ts

```diff
@@ -0,0 +1,11 @@
+const getNumberOfDays = (start: Date, end: Date): number => {
```

Comment:
function relativeDaysUntilDate(dateString: string): string {
  const now = Date.now();
  const futureDate = new Date(dateString);
  const timeDifference = futureDate.getTime() - now;
  const dayDifference = Math.ceil(timeDifference / (1000 * 3600 * 24));
  const formatter = new Intl.RelativeTimeFormat('en', {
    numeric: 'auto',
  });

  return formatter.format(dayDifference, 'day');
}

As another option, not sure If this better or not

Reply:
both are acceptable